packages/modular-scripts/src/cli.ts

```diff
@@ -12,7 +12,7 @@ import {
 } from 'change-case';
 import prompts from 'prompts';
 import resolveAsBin from 'resolve-as-bin';
-
+import buildPackage from './build';
```

Comment:
Build does a lot of top level work - so potentially we want to hide this import or change the top level work done?

Reply:
I didn't understand this comment, could you clarify what you mean? Do you want me to move build() into this file?

Reply:
My main concern is that there are import side-effects for `build.ts` e.g. [1] / [2] which do a lot of FS operations. On a machine with a slow-FS (e.g. network drives) this could slow down the startup of the cli for all operations. I think we should either:

1. Remove the globals from the `build.ts` script and have them as locals for the build operations, or. 
2. Lazily import `build.ts` so that the initialisation only happens when it's needed. 

[1] https://github.com/jpmorganchase/modular/pull/195/files#diff-a3c185d299c664d2bc8bbc2bbade48ca2a96a5ed03c585d268e5290e97016c0aR77
[2] https://github.com/jpmorganchase/modular/pull/195/files#diff-a3c185d299c664d2bc8bbc2bbade48ca2a96a5ed03c585d268e5290e97016c0aR96

Reply:
Good points. Ok lemme rewrite this.

Reply:
Ok this import is now deferred till the time it's actually called.